redhat-developer/developers.redhat.com
# Drupal console in dev

A few tweaks to make drupal console available via `./run-drupal-connect.sh`.

To test: 
* `./run-drupal.sh to rebuild drupal`
* `./run-drupal-connect.sh`
* `drupal` should be available and work.

Note: because the home directory of the user in the docker container is `/var/www/drupal`, which is read only, you'll get some error spam as the utility tries to create a `.console` directory in the home directory. This is harmless but probably slows it down a bit. You can make the errors go away by setting some new home directory once connected:

`mkdir -p /tmp/home; HOME=/tmp/home`
